cardano-node/src/Cardano/Node/Protocol/Shelley.hs

```diff
@@ -181,9 +181,9 @@ readLeaderCredentialsSingleton ProtocolFilepaths {shelleyKESFile = Nothing} =
      left KESKeyNotSpecified
 
 opCertKesKeyCheck
-  :: FilePath
+  :: File () In
```

Comment:
We can use `()` for when we previously used `FilePath`.

This means we're not tracking the content type of the file yet.  We can add this later.